static/app/components/events/interfaces/spans/spanTree.tsx

```diff
@@ -214,8 +214,8 @@ class SpanTree extends React.Component<PropType> {
               treeDepth={treeDepth}
               continuingTreeDepths={continuingTreeDepths}
               spanNumber={spanNumber}
-              spanGrouping={payload.spanGrouping as EnhancedSpan[]}
-              toggleSpanGroup={payload.toggleSpanGroup as () => void}
+              spanGrouping={payload.spanNestedGrouping as EnhancedSpan[]}
+              toggleSpanGroup={payload.toggleNestedSpanGroup as () => void}
```

Comment:
Should we update the names of these props on `SpanGroupBar` as well so they match?

Reply:
@0Calories those are the next set of renames I'd like to do on the follow up pull request. I wanted to keep the scope of this set of changes limited to just the mobx classes.